### ISSUE TITLE
Do you change any version of packages when running on torch1.13.1?

### ISSUE THREAD

**Issue by SainingZhang** [Open3DVLab/NeuRodin#3]

**Comment by yyfz** [comment]
Hi! We don't modify package versions when working with torch 1.12.1 and cuda 11.3. We have tested the environment installation commands provided in the README.md on an RTX3090 and RTX4090.

If you intend to use torch 1.13.1, the crucial step is to verify that the cuda version aligns with the torch version. Also, make sure that tinycudann works correctly in your environment.

**Comment by SainingZhang** [comment]
Thank you very much! Wonderful work! I wonder how many iterations you trained for outdoor scenes for both stage 1 and 2?

**Comment by yyfz** [comment]
Thank you for your interest. We trained the model for 300,000 iterations for each stage.

**Comment by SainingZhang** [comment]
Thank you. Do you use tinycudann 1.7?

**Comment by yyfz** [comment]
> Thank you. Do you use tinycudann 1.7?

 Yes, `pip install git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch` will install tinycudann==1.7.
